From 4b23c742e46ed05ffc281147ea75e0705e2164df Mon Sep 17 00:00:00 2001
From: Faouaz TENOUTIT <faouaz.tenoutit@intel.com>
Date: Tue, 24 Feb 2015 11:46:57 +0100
Subject: [PATCH 24/24] ASoC: Intel: Add support for Malata (A82i)

The audio codec can't be correclty identified on the Malata (A82i) variant,
due to the missing DMI string id.

This patch will add the missing DMI string identifier.

Issue: IRDA-2800

Change-Id: Ic9cc4b223facf25a2cc4d76b55a1d3538d0a4482
Signed-off-by: Faouaz TENOUTIT <faouaz.tenoutit@intel.com>
---
 sound/soc/intel/board/byt_cr_board_configs.h | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/sound/soc/intel/board/byt_cr_board_configs.h b/sound/soc/intel/board/byt_cr_board_configs.h
index 6366208..5b05a5a 100644
--- a/sound/soc/intel/board/byt_cr_board_configs.h
+++ b/sound/soc/intel/board/byt_cr_board_configs.h
@@ -57,6 +57,7 @@ enum board_id_rt5640 {
 	RT5640_MRD7,
 	RT5640_T100,
 	RT5640_MALATA,
+	RT5640_MALATA_A82I,
 	RT5640_CHIPHD,
 };
 
@@ -143,6 +144,14 @@ static const struct dmi_system_id dmi_system_ids0[] = {
 		},
 		.driver_data = (void *)&board_configs0[RT5640_MALATA],
 	},
+	[RT5640_MALATA_A82I] = {
+		.ident = "A82i",
+		.matches = {
+			DMI_MATCH(DMI_BOARD_NAME, "A82i"),
+			DMI_MATCH(DMI_BOARD_VERSION, "1"),
+		},
+		.driver_data = (void *)&board_configs0[RT5640_MALATA],
+	},
 	[RT5640_CHIPHD] = {
 		.ident = "CHIPHD",
 		.matches = {
-- 
1.9.1

